New package: DifferentialRiccatiEquations v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: DifferentialRiccatiEquations
- Repository: https://github.com/mpimd-csc/DifferentialRiccatiEquations.jl
- Created by: @jonas-schulze
- Version: v0.3.0
- Commit: 80527fa2cf6917dc50661f4d6556e6a06559b522
- Reviewed by: @jonas-schulze
- Reference: https://github.com/mpimd-csc/DifferentialRiccatiEquations.jl/issues/1#issuecomment-1988572406
- Description: Solvers for autonomous matrix-valued generalized differential Riccati equations (GDREs) with low-rank solutions.
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835982b43aca326e220112ce6e7cc3474753321584f17791a975abef2fcfc51c78d53fe0e6e4d8d2fbac7cf33a2b7d4287f1f7db5994c463a05121897da15af17310c2a20c48b3d70a7e9fe2dcfb724d6ef8a645f92127700e2bc9c7e5be30262c97624d093be6b530238c13f49104240dc7c75b32dcbf378006b6225a785aa220f864c3435dae28d0131468a7cd135d5f4f7 -->

jonas-schulze commented:
One of the checks appears to be a false negative (fatal: unable to access 'https://github.com/JuliaRegistries/General.git/': Failed to connect to github.com port 443 after 32 ms: Connection refused). Did this failure prevent this PR from being (auto-) merged?

GunnarFarneback commented:
[noblock]

You could try to register the same commit again and see if that restarts the tests. The failed check is clearly a transient problem. The waiting time won't be reset from a re-registration.

JuliaRegistrator committed:
New package: DifferentialRiccatiEquations v0.3.0

UUID: 9819f967-4574-4d9d-b214-002eeba273d9
Repo: https://github.com/mpimd-csc/DifferentialRiccatiEquations.jl.git
Tree: f635b38a4d310c0501d1f9dca006f4e246f41270

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

jonas-schulze commented:
[noblock]

Thanks, @GunnarFarneback! Let's hope for the best. :crossed_fingers:
